include/wifi/WiFiManager.h

```diff
@@ -64,4 +65,8 @@ namespace OpenShock::WiFiManager {
 
   /// @brief Runs the WiFiManager loop
   void Update();
+
+  /// @brief Gets the vector of discovered WiFi networks
+  /// @return Vector of discovered WiFi networks
+  std::vector<WiFiNetwork>& GetDiscoveredWiFiNetworks();
```

Comment:
This should not return a reference of the vector, this might change after we called this function so we want to create a copy of it instead.